cmd/bosun/expr/logstash.go

```diff
@@ -122,6 +122,12 @@ func (e *LogstashElasticHosts) GenIndices(r *LogstashRequest) (string, error) {
 	if err != nil {
 		return "", err
 	}
+	// Short-circut when using concrete ES index name
+	if len(r.IndexRoot) > 0 {
```

Comment:
This and the line below can be reduced to `if strings.HasSuffix(r.IndexRoot, "/")`.